slurm.spec

```diff
@@ -325,6 +325,7 @@ notifies slurm about failed nodes.
 	%{?_without_x11:--disable-x11} \
 	%{?_with_ucx} \
 	%{?_with_cflags}
+	%{?_with_x11}
```

Comment:
@itkovian how does this relate to the `without_x11` above? and this is also missing a `\` on previous line